definitions/private-law/json/AuthorisationCaseField/ManageOrders/AuthorisationCaseField.json

```diff
@@ -1006,5 +1006,19 @@
     "CaseFieldID": "orderCollection",
     "UserRole": "caseworker-privatelaw-systemupdate",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "09/05/2022",
+    "CaseTypeID": "PRLAPPS",
+    "CaseFieldID": "magistrateLastName",
```

Comment:
Don't we need permission for judge and la for this field?

Reply:
Comments added.